lib/cmd/fh3/credential/create.js

```diff
@@ -0,0 +1,105 @@
+/* globals i18n */
+var fhreq = require('../../../utils/request.js');
+var fs = require('fs');
+
+module.exports = {
+  'desc' : i18n._('Create a credential bundle'),
+  'examples' : [{
+    cmd : 'fhc credential create --bundleName=<bundle-name> --bundleType=<bundle-type> --platform=<platform> --env=<env> --privateKey=<private-key> --certificateAndroid=<certificate-android>',
+    desc : 'Create a credential for Android',
+  },
+  {
+    cmd : 'fhc credential create --bundleName=<bundle-name> --bundleType=<bundle-type> --platform=<platform> --env=<env>  --privateKey=<private-key> --certificateIos=<certificate-ios> --provisioning=<provisioning>',
+    desc : 'Create a credential for IOS',
+  }],
+  'demand' : ['bundleName', 'platform'],
+  'alias' : {},
+  'describe' : {
+    'bundleName' : i18n._('Name for the credential which you want create.'),
+    'bundleType': i18n._("FThe default value is 'release'. Option valid just for IOS platform. Options :['release','debug', 'distribution']"),
+    'platform': i18n._("Platform Options : [android, ios]"),
+    'env': i18n._("Default value is all. Environment ID which you want create the credential."),
+    'privateKey': i18n._("Param valid for IOS and Android platform. A Private Key is a file whose contents are secret to the owner. During the App building process, the App is digitally signed using this key. Please make sure that Private Key is password protected. Path for the private key file (E.g --privateKey:/Users/user/rivateKey)"),
```

Comment:
👍 It is the same text that has in the UI. It was changed for just  "Path for the private key file (E.g --privateKey=/Users/user/privateKey)". I agree, too big and the information is not helpful.